Sources/WebDriver/Requests.swift

```diff
@@ -524,6 +524,41 @@ public enum Requests {
         }
     }
 
+    // https://www.selenium.dev/documentation/legacy/json_wire_protocol/#sessionsessionidlocation
+    public enum SessionGeoLocation {
+        public struct Post: Request {
+            public var session: String
+            public var latitude: Int
```

Comment:
These should be Doubles since values will definitely be more granular than integers.